File: Typescript/Practice/26072022/easy/easy_countInstances.py
#!/usr/bin/env python

def char_count(str1, str2):
  counter = 0
  for c in str2:
    if c == str1:
      counter += 1
  return counter

print(char_count("a", "edabit"))
print(char_count("c", "Chamber of secrets"))
print(char_count("b", "big fat bubble"))

File: Typescript/Practice/26072022/easy/easy_countInstances.ts
#!/usr/bin/env npx ts-node

function charCount (str1: string, str2: string): number {
  let counter: number = 0;
  for (let char of str2) {
    if (str1 === char) {
      counter++
    };
  };
  return counter;
};

console.log(charCount("a", "edabit"));
console.log(charCount("c", "Chamber of secrets"));
console.log(charCount("b", "big fat bubble"));

